### PR TITLE
Include additional verbs to can-i

### PR DESCRIPTION
**Pull request by micahhausler**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Add missing verbs to `kubectl auth can-i` command. Both verbs are valid, but were not included in the existing list.

#### Which issue(s) this PR fixes:

N/A

#### Special notes for your reviewer:

N/A

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

/sig auth

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ var (
 		# List all allowed actions in namespace "foo"
 		kubectl auth can-i --list --namespace=foo`)
 
-	resourceVerbs       = sets.NewString("get", "list", "watch", "create", "update", "patch", "delete", "deletecollection", "use", "bind", "impersonate", "*", "approve")
+	resourceVerbs       = sets.NewString("get", "list", "watch", "create", "update", "patch", "delete", "deletecollection", "use", "bind", "impersonate", "*", "approve", "sign", "escalate")
```
Comment (enj) [inline staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go:108]: Do we want to add `attest` even though cluster trust bundles are alpha?

Comment (micahhausler) [reply staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go:108]: I can feature-flag add it :)

Comment (micahhausler) [reply staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go]: Ah, can't import `k8s.io/kubernetes` in `staging/` packages

**Pull request merged**